<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>828</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Photo Manager</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>80</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Compare Path</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmpLoadBtn">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>70</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="baseImgView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>531</width>
     <height>281</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="basePath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="baseLoadBtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>70</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="cmpImgView">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>400</y>
     <width>551</width>
     <height>281</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>20</width>
     <height>621</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>531</width>
     <height>241</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>529</width>
      <height>239</height>
     </rect>
    </property>
    <widget class="QListView" name="baseListView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>241</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLineEdit" name="comparePath">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>70</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>160</y>
     <width>551</width>
     <height>241</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>549</width>
      <height>239</height>
     </rect>
    </property>
    <widget class="QListView" name="cmpListView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>551</width>
       <height>241</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Base Path</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>710</y>
     <width>1111</width>
     <height>107</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1109</width>
      <height>105</height>
     </rect>
    </property>
    <widget class="QTextBrowser" name="resultText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1111</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="cmpBtn">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>20</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Compare</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeSelecFilesBtn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>120</y>
     <width>201</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove selected files</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CmpSelecAllBtn">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>120</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BaseSelecAllBtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>baseLoadBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>baseLoadBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmpLoadBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>cmpLoadBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1104</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>1082</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmpBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>cmpBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>821</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>945</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CmpSelecAllBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>cmpSelecAllBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1039</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>977</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BaseSelecAllBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>baseSelecAllBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>434</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeSelecFilesBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>removeSelecFilesBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1006</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>707</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>baseLoadBtnClicked()</slot>
  <slot>cmpLoadBtnClicked()</slot>
  <slot>cmpBtnClicked()</slot>
  <slot>cmpSelecAllBtnClicked()</slot>
  <slot>baseSelecAllBtnClicked()</slot>
  <slot>removeSelecFilesBtnClicked()</slot>
 </slots>
</ui>
